public/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
+require '../vendor/autoload.php';
+Sample\Index::main();
```

Comment:
For the Router not to use the root folder as reference you would have to pass __DIR__ to the main method and then into the router. Or (better yet) receive a router instance in the main method and initialize it here.
Otherwise the url parsing simply breaks.